### PR TITLE
removed extra word and added paragraph

### PR DESCRIPTION
**Pull request by prjseal**
removed extra word and added paragraph about client dependency and running without debugging

### CODE REVIEW

File: Tutorials/Creating-a-Custom-Dashboard/index.md [hunk 1]
```diff
@@ -130,6 +130,8 @@ This stylesheet will now be loaded and applied to your dashboard. Add images and
 
 *__Note:__ One caveat is the package.manifest file is loaded into memory when Umbraco starts up, so if you are adding a new stylesheet or JavaScript file you will need to start and stop your application for it to be loaded.*
 
+If the title doesn't change colour, you may be running the site without debugging and therefore client dependency will be serving the javascript and css in one file which gets cached. To clear that cached file and get it to load in the new javascript and styles, you need to increment the the version number in the ClientDependency.config file and press save. Now you can reload the page and see the colourful title.
```
Comment (dampee) [inline Tutorials/Creating-a-Custom-Dashboard/index.md]: A few remarks: 

* I would mark "Client Dependency" with capitals and maybe add a link to make clear it is another component.
* "serving the javascript an css in one file" sounds like javascript and css is bundled in one file, which is uhm... not.  They are each bundled in one file.  :-)  Could you rephrase?
* I would add a link to the clientDependency config file documentation

Comment (prjseal) [reply Tutorials/Creating-a-Custom-Dashboard/index.md]: I've implemented these suggestions

**Comment by dampee** [comment]
Thanks a lot @prjseal
